WireUI/Sources/WireIndividualToTeamMigrationUI/Models/TeamPlanFeature.swift

```diff
@@ -27,3 +27,33 @@ public struct TeamPlanFeature: Hashable, Identifiable, Sendable {
         self.description = description
     }
 }
+
+extension Array where Element == TeamPlanFeature {
+    static var features: [TeamPlanFeature] {
```

Comment:
The content in `TeamPlanSelectionPreview` is now a duplicate of this.
Does it make sense to remove this list from the preview file?

Reply:
Very good point, thanks! Removed.